staker/getter.gno

```diff
@@ -0,0 +1,181 @@
+package staker
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/grc/grc20"
+	u256 "gno.land/p/demo/uint256"
+	"gno.land/p/demo/ufmt"
+)
+
+// region getters
+
+// GetTotalStaked returns the total aount of tokens staked in the contract.
+func GetTotalStaked() string {
+	return gStaker.totalStaked.ToString()
+}
+
+// GetDelegationAmount returns the amount of tokens delegated to a given address.
+func GetDelegationAmount(delegate string) string {
+	delegateAddr := std.Address(delegate)
+	if !delegateAddr.IsValid() {
+		panic(ufmt.Sprintf("invalid address: %s", delegate))
+	}
+
+	amount, exists := gStaker.delegations[delegateAddr]
+	if !exists {
+		return "0"
+	}
+
+	return amount.ToString()
+}
+
+// GetAllDelegations returns a JSON string of all delegations,
+// with addresses as keys and amounts as values.
+func GetAllDelegations() string {
+	result := make(map[string]string)
+	for addr, amount := range gStaker.delegations {
+		result[addr.String()] = amount.ToString()
+	}
+	return marshalDelegationsJSON(result)
```

Comment:
I don't think it's good idea to marshal manually.
If item has been added or removed, it's just too much TODO work.

Let's use `demo/json`

Reply:
fixed https://github.com/gnoswap-labs/governance/pull/17/commits/00f198e62ab5ef8f260d9e6b16045979a5ffbd58